Makefile

```diff
@@ -460,3 +465,19 @@ else
 	mkdir -p $(dir $@)
 	uuidgen | tr '[:upper:]' '[:lower:]' > $@
 endif
+
+
+###########
+## Tools ##
+###########
+
+# Local Node Scanner
+.PHONY: local-nodescanner
+local-nodescanner:
+	@echo "+ $@"
+	GOOS=linux GOARCH=amd64 go build -trimpath -o ./tools/bin/local-nodescanner ./tools/local-nodescanner
```

Comment:
Should we build this inside our `$(BUILD_IMAGE)`?

Reply:
I wanted to keep the tool rather isolated, but if you think we should use the build image, I'm happy to move that over into it.

Reply:
By isolation, do you mean minimizing dependencies? If yes, I think it's a very good goal.

If the primary goal of the tool is to aid development and local testing, I think it is OK not to use the build image to build it. Plan to have this evolve into something we can leverage in CI. Using the builder image might be necessary to ensure the built environment doesn't affect the final binary.

Reply:
Thanks for the feedback! I've introduced a new dockerized build that is based on the builder image. Let me know what you think! 
See https://github.com/stackrox/scanner/pull/1164/commits/07e33824d9936c620f60d2a017297ec5a06fd3cc